packages/runtime/runtime-utils/src/summarizerNode/summarizerNode.ts

```diff
@@ -224,6 +224,14 @@ export class SummarizerNode implements IRootSummarizerNode {
                 this.refreshLatestSummaryFromPending(proposalHandle, maybeSummaryNode.referenceSequenceNumber);
                 return { latestSummaryUpdated: true, wasSummaryTracked: true };
             }
+
+            this.defaultLogger.sendTelemetryEvent({
+                eventName: "PendingSummaryNotFound",
+                proposalHandle,
+                summaryRefSeq,
+                size: this.pendingSummaries.size,
+                haveHandle: this.pendingSummaries.size > 0 ? this.pendingSummaries.keys[0] : undefined,
```

Comment:
What does `haveHandle` represent?

Reply:
HaveHandle is an existing column I'm making use of to identify the proposal we have pendingsummary for (local list).
I'm also adding the referenceSequenceNumber from that proposal and the current one. Let me know what you think :)